geomesa-spark/geomesa_pyspark/src/main/python/setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='geomesa_pyspark',
+    author='Tom Kunicki, Gerard C. Briones',
+    version='${python.version}',
+    author_email='tkunicki@ccri.com, gbriones@ccri.com',
```

Comment:
Is this typical?  

Is there a way to indicate geomesa.org as the Website?  And/or the geomesa-user list for questions?